font-inspector.html

```diff
@@ -161,22 +163,32 @@ <h1>Free Software</h1>
 }
 
 function displayNames(names) {
-    var html = '';
-    properties = sortKeys(names);
-    for (var i = 0; i < properties.length; i++) {
-        var property = properties[i];
-        html += '<dt>'+escapeHtml(property)+'</dt><dd>';
-        var translations = names[property];
-        var langs = sortKeys(translations);
-        for (var j = 0; j < langs.length; j++) {
-            var lang = langs[j];
-            var esclang = escapeHtml(lang);
-            html += '<span class="langtag">' + esclang
-                + '</span> <span class="langname" lang=' + esclang + '>'
-                + escapeHtml(translations[lang]) + '</span> ';
+    let html = '';
+    ['unicode', 'macintosh', 'windows'].forEach( platform => {
+        platformNames = names?.[platform];
```

Comment:
where is the let declaration, we don't want these variables to be global.

Reply:
Good catch, and it shows why code review is so important! That's what happens when you frequently switch between JS and PHP development... 😉